### DISCUSSION TITLE
Feature Request: concat_list: support more than two arguments

### DISCUSSION THREAD

**Discussion by NickCrews** [duckdb/duckdb#14685] [Ideas]
When working with strings, [concat(string, ...)](https://duckdb.org/docs/sql/functions/char.html#concatstring-) is nice because it works with N strings. It would be great if [list_concat(list1, list2)](https://duckdb.org/docs/sql/functions/list#list_concatlist1-list2) also worked with >=2 args

**Comment by Mytherin** [comment]
This is actually already implemented in feature - see https://github.com/duckdb/duckdb/pull/14443

**Reply by NickCrews** [reply]
Thanks! And sorry, I should have searched issues. Want me to make a docs PR, or do we only do that once the feature is released? Maybe I make one and mark as draft, then we let it sit until release?
